src/pages/resultsView/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -32,7 +32,7 @@ export const OrderedAlterationLabelMap: Record<keyof IAlterationCountMap, string
 const alterationToColor: Record<keyof IAlterationCountMap, string> = {
     mutated: "#008000",
     amp: "#ff0000",
-    homdel: "rgb(0,0,255)",
+    homdel: "#8fd8d8",
```

Comment:
do we want to sync this with the oncoprint colors eg https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/components/oncoprint/geneticrules.ts#L73

Reply:
uh, yes

Reply:
but i think we should revisit this after this PR

Reply:
sounds good